presto-hive/pom.xml

```diff
@@ -122,6 +122,11 @@
             <artifactId>aws-java-sdk-s3</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>com.amazonaws</groupId>
+            <artifactId>aws-java-sdk-dynamodb</artifactId>
```

Comment:
What is this for?

Reply:
It's required by `org.apache.hadoop.fs.s3a.S3AFileSystem`